migrations/20210927094004_payment/migration.sql

```diff
@@ -0,0 +1,283 @@
+ADD COLUMN     "updated_at" TIMESTAMPTZ(6);
+
+-- AlterTable
+ALTER TABLE "people" ADD COLUMN     "personal_number" TEXT,
+ADD COLUMN     "vault_id" UUID,
+ALTER COLUMN "address" SET DATA TYPE VARCHAR(100);
+
+-- DropEnum
+DROP TYPE "Currency";
+
+-- CreateTable
+CREATE TABLE "companies" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "company_name" VARCHAR(100) NOT NULL,
+    "companyNumber" TEXT NOT NULL,
+    "legal_person_name" TEXT,
+    "country_code" CITEXT,
+    "city_id" UUID,
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "companies_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "benefactors" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "person_id" UUID NOT NULL,
+    "customer_id" TEXT,
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "benefactors_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "vaults" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "type" "vault_type" NOT NULL,
+    "amount" DECIMAL(65,30) NOT NULL DEFAULT 0.0,
+    "currency" "currency" NOT NULL DEFAULT E'BGN',
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "vaults_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "transactions" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "type" "transaction_type" NOT NULL,
+    "status" "transaction_status" NOT NULL DEFAULT E'initial',
+    "provider" "payment_provider" NOT NULL DEFAULT E'none',
+    "target_vault_id" UUID NOT NULL,
+    "customer_id" TEXT NOT NULL,
+    "payment_intent_id" TEXT NOT NULL,
+    "payment_method_id" TEXT NOT NULL,
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "transactions_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "subscriptions" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "status" "subscription_status" NOT NULL,
+    "vault_id" UUID NOT NULL,
+    "person_id" UUID NOT NULL,
+    "subscription_id" TEXT NOT NULL,
+    "customer_id" TEXT,
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "subscriptions_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "transfers" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "type" "transfer_type" NOT NULL,
+    "amount" DECIMAL(65,30) NOT NULL,
+    "reason" VARCHAR(100) NOT NULL,
+    "source_vault_id" UUID NOT NULL,
+    "target_vault_id" UUID NOT NULL,
+    "target_date" TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP,
+    "created_at" TIMESTAMPTZ(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMPTZ(6),
+
+    CONSTRAINT "transfers_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "withdraws" (
```

Comment:
Great catch for the naming `withdrawal` :+1: 

About where the money will be going I think you're observing an old version of the schema. Let's discuss in the `schema.prisma` file instead of `sql` files.

In the following schema I've renamed the column `source_vault_id` already to `vault_id` as we have only one direction money flow with this object `Withdrawal`

https://github.com/podkrepi-bg/api/blob/e6715f09d07cd7f7b03b9979a308b573ed3250fa/schema.prisma#L299-L314